website/docs/patterns/button-organization/partials/guidelines/alignment.md

```diff
@@ -6,14 +6,14 @@ Alignment refers to how buttons and actions are aligned relative to the containe
 
 When considering internationalization (i18n), **start** and **end** are suitable substitutes for **left** and **right** alignment to account for languages that are read right-to-left ([RTL](https://developer.mozilla.org/en-US/docs/Glossary/RTL)).
 
-![Button alignment options](/assets/patterns/button-organization/alignment-methods.png)
+![Button alignment options](/assets/patterns/button-organization/alignment-methods.png =770x*)
 !!!
 
 ### Left alignment
 
 Actions are aligned to the left of the container or page. A common example of this alignment method is in [Modals](/components/modal) and page-level forms.
 
-![Left alignment example](/assets/patterns/button-organization/using-left-alignment.png)
+![Left alignment example](/assets/patterns/button-organization/using-left-alignment.png =836x*)
```

Comment:
This image looks significantly bigger compared with the others. Consider making the artboard 770px wide and apply @didoo's suggestion or updating the width value from `836` to `690` to make it more proportional.

Reply:
Yes good catch on this one! I missed it in the rest of the image resizing.